src/splits/splitHelpers.ts

```
@@ -555,3 +583,286 @@
+
+    const viemWalletClient = createWalletClient({
+      account,
+      chain: chain,
+      transport: http(CHAIN_PUBLIC_RPC_URL[chainId])
+    });
+    return new SplitsClient({
+      publicClient: client,
+      walletClient: viemWalletClient,
+      chainId,
+    });
+  };
```

Biome (1.9.4)

[error] This code is unreachable

... because either this statement will return from the function, ...

... this statement will throw an exception, ...

... or this statement will return from the function beforehand


(lint/correctness/noUnreachable)